nab/detectors/skyline/skyline_detector.py

```diff
@@ -11,6 +11,13 @@
 
 
 class SkylineDetector(AnomalyDetector):
+  """ Detects anomalies using Etsy Skyline's ensemble of algorithms from
+  https://github.com/etsy/skyline/blob/master/src/analyzer/algorithms.py.
+  The scoring of this detector differs slightly from Skyline. Skyline
+  relies on consensus of algorithms in the ensemble based on majority voting to
+  classify a metric as anomalous whereas this detector combines the votes of
+  all algorithms and assigns an average anomaly score.
```

Comment:
This is a little confusing. Please describe that each algorithm in the ensemble gives a binary vote for the data record. The original implementation used a majority voting scheme to classify a record as anomalous. Here we improve the detector's performance by...